README.MD

```diff
@@ -0,0 +1,132 @@
+# Riot Developer Example App
+This project is a consumption example of [Riot Games Developer API](https://developer.riotgames.com/apis) (RGAPI).
+
+
+# Project Setup
+To setup this project locally a few steps are needed, they are:
+- [Configure environment variables](#configure-environment-variables)
+- [Add rule for localhost domain (only for oauth2 authentication)]
+(#add-rule-for-localhost-domain-only-for-oauth2-authentication)
+- [Install python dependencies](#install-python-dependencies)
+
+## Install python dependencies
+You should install some dependencies to use the app: 
+- pip install Flask
+- pip install python-dotenv
+- pip install dataclasses
+- pip install numpy
+
+## Configure environment variables
+### API Key
+You can get the API Key after sign in at [Riot Games Developer Portal](https://developer.riotgames.com/login). This token will be used as environment variable `APP_KEY`.
+
+### Get credentials for oauth2
+For oauth2 authentication are needed two credentials, they are `CLIENT_ID` and `CLIENT_SECRET` and will be used respectively for environment variables `RSO_CLIENT_ID` and `RSO_CLIENT_SECRET`. Provided by (https://developer.riotgames.com/)
+
+### Create .env file
+Copy file `.env.example` to a new file named `.env`.
+
+### Update .env file
+Set the environment variables with the credentials created in the last steps. For example:
+```
+APP_PORT=5000
+APP_CALLBACK_PATH = <YOUR-CALLBACK-PATH>
+RSO_CLIENT_ID=<YOUR-CLIENT-ID>
+RSO_CLIENT_SECRET=<YOUR-CLIENT-SECRET>
+APP_KEY=<YOUR-API-KEY>
+
+```
+
+## Add rule for localhost domain (only for oauth2 authentication)
+To use oauth2 authentication redirect flows is needed to match your app hostname with the `redirect_uri` provided when [getting credentials for oauth2](#get-credentials-for-oauth2).
+
+### Windows
+Open hosts file `C:\Windows\System32\Drivers\etc\hosts` \
+Add your hostname to localhost ip
+```
+127.0.0.1  YOUR-REDIRECT-URI
```

Comment:
May want to put a sample one in here so people know the format, for instance not prefixing `http` and stuff like that, maybe `local.mysite.com` or something